conf/messages.en

```diff
@@ -437,7 +437,7 @@ departure.declaration.errors.message.hyperlink = Create another departure declar
 cancellation.notification.errors.message.title   = Cancellation errors
 cancellation.notification.errors.message.heading = Cancellation errors
 cancellation.notification.errors.message.noerrors = There are one or more errors with the cancellation of departure declaration {0}.
```

Comment:
I think this can be deleted if it's not being used

Reply:
I think this is being used Reece in CancellationNotificationErrorsP5ViewModel, so I am not sure if we can delete it... 
    if (noErrors) {
      messages("cancellation.notification.errors.message.noerrors", lrn)
    }

Reply:
Hmm true, maybe leave them in for now. There is two tickets to address these changes where this should get taken care of anyways. (CTCP-3284, CTCP-3289)

Reply:
Resolved, will been taken care in another ticket